internal/operands/metrics/resources.go

```diff
@@ -16,6 +16,129 @@ const (
 	componentAlertLabelValue = "ssp-operator"
 )
 
+// RecordRulesDesc represent SSP Operator Prometheus Record Rules
+type RecordRulesDesc struct {
+	Name        string
+	Expr        intstr.IntOrString
+	Description string
+}
+
+// RecordRulesDescList returns SSP Operator Prometheus Record Rules
```

Comment:
As far as I understand - this comment does not give an accurate description, the object below does not return things. It is a list that holds objects which are partial representations of the recording rules. I would just remove this comment, the name is good enough.